cellxgene_gateway/filecrawl.py

```diff
@@ -48,7 +48,7 @@ def make_entry(el):
                 "type": "neither",
             }
 
-    return [make_entry(x) for x in os.listdir(path)]
+    return [make_entry(x) for x in sorted(os.listdir(path))]
```

Comment:
I realize there is some redundant code here, could you reuse the all_entries variable here? 
```suggestion
    return [make_entry(x) for x in all_entries]
```